doc/userguide.rst

```diff
@@ -90,14 +90,14 @@ the package. More specifically, the subfolders of `examples/` contain:
   the forces were altered in a way to have both the forces and the potential
   smoothly reaching zero at the cut-off distance. For each choice of interaction
   function, the same simulation was repeated five times, with each new simulation
-  halving the integration time step compared to the previous one.
+  having the integration time step compared to the previous one.
 
 For each example, the subfolders for individual simulations contain the following files:
 
 * `start.gro`: the starting configuration of the molecules/atoms
 * `system.top`: the topology of the system
 * `system.mdp`: the GROMACS input file
-* `mdout.mdp`: the GROMACS input file obtained from `gmx grompp`
+* `mdout.mdp`: the GROMACS output file obtained from `gmx grompp`
```

Comment:
Maybe we can make that even more clear. How about "GROMACS parameters file obtained from"?

Reply:
Yeah I think that works better. Changes have been made.